vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -43,10 +42,6 @@ def textual_hostname
     {:label => "Hostname", :value => @ems.hostname}
   end
 
-  def textual_ipaddress
-    {:label => "IP Address", :value => @ems.ipaddress}
```

Comment:
I think what we needed here was just:

    {:label => "IP Address", :value => @ems.ipaddress} unless @ems.ipaddress.nil?